### DISCUSSION TITLE
How to add super reactions using a discord.py bot

### DISCUSSION THREAD

**Discussion by imeesa** [Rapptz/discord.py#9729] [Q&A] [answered]
I have a discord bot (using discord.py), and would like it to be able to super react to messages. As of right now, the discord.Message.add_reaction function does not have an option to add a *super* reaction. This ability would be useful for circumstances where the bot would normally react, but it is a more important occasion.

**Comment by Rapptz** [comment]
Bots can't add super reactions.

**Reply by imeesa** [reply]
discord.py still allows for selfbots though.

**Reply by IAmTomahawkx** [reply, marked as answer]
discord.py has no selfbotting capabilities as of 2.0, nor is it a supported usecase

**Reply by MCausc78** [reply]
> discord.py still allows for selfbots though.

discord.py doesn't allow logging as non-Bot account, and self-bots are against tos
